from gurobipy import Model, GRB, quicksum, GurobiError

try:
    modelo = Model('AldeaMatematica')
    
